driver/vhci/vhci_vpdo.c

```diff
@@ -266,7 +268,7 @@ setup_vpdo_compat_ids(pusbip_vpdo_dev_t vpdo, PIRP irp)
 	RtlStringCchPrintfW(ids_compat, 33, L"USB\\Class_%02hhx&SubClass_%02hhx&Prot_%02hhx", vpdo->usbclass, vpdo->subclass, vpdo->protocol);
 	RtlStringCchPrintfW(ids_compat + 33, 25, L"USB\\Class_%02hhx&SubClass_%02hhx", vpdo->usbclass, vpdo->subclass);
 	RtlStringCchPrintfW(ids_compat + 58, 13, L"USB\\Class_%02hhx", vpdo->usbclass);
-	if (vpdo->inum > 1 || IS_IAD_DEVICE(vpdo)) {
```

Comment:
Is there any reason for combining a vpdo->inum condition with IS_ZERO_CLASS() ?
This prevents a non-zero and non-IAD device with multiple interfaces to be a composite.

Reply:
so far I never worked with such device :) except these two groups of CLS/SUBCLS/PRO =0 and IAD. I think it might be named with other category. Please correct me if I am wrong.

Reply:
Ok, I admit that dropping vpdo->inum condition is a reasonable choice.